TESTING.md

```diff
@@ -11,72 +11,82 @@ Recipeezy
 
 # Automated Test Cases:
 
+PRECONDITION TO RUN ALL TESTS:
+1. create a new environment
+2. run `pip install -r requirements.txt` in environment to get requirements
 ---
 Use case name
     Verify login with valid user name and password
 Description
-    Test the Google login page
+    Test for logging in only. This tests for access to the User database and
+    return to the HOME page.
 Pre-conditions
-    User has valid user name and password
+    No preconditions
 Test steps
-    1. Navigate to login page
-    2. Provide valid user name
-    3. Provide valid password
-    4. Click login button
+    1. run app using `python recipeezy.py`
+    2. open browser and delete cookies
+    3. go to the url given by app
+    4. click on CREATE
+    5. In Username, type "New_User"
+    6. In Password, type "Password"
```

Comment:
These are wrong. username is new_user and password is password. I will change them after merge